src/Tasks/Microsoft.NET.Build.Tasks/ResolveTargetingPackAssets.cs

```diff
@@ -46,6 +46,8 @@ public class ResolveTargetingPackAssets : TaskBase
         [Output]
         public ITaskItem[] UsedRuntimeFrameworks { get; set; }
 
+        private static readonly bool s_AllowCacheLookup = System.Environment.GetEnvironmentVariable("DOTNET_SDK_ALLOWTARGETINGPACKCACHING") != "0";
```

Comment:
It doesn't look like there are a lot of SDK escape hatches like this. Does this look ok?